lms/tasks/email_digests.py

```diff
@@ -14,14 +13,13 @@
     User,
 )
 from lms.services import DigestService
-from lms.services.digest import SendDigestsError
 from lms.tasks.celery import app
 
 LOG = logging.getLogger(__name__)
 
 
-@app.task(acks_late=True, bind=True, max_retries=2)
-def send_instructor_email_digest_tasks(self, *, batch_size, h_userids=None):
+@app.task(acks_late=True, autoretry_for=(Exception,), max_retries=2, retry_backoff=3600)
```

Comment:
See https://docs.celeryq.dev/en/stable/userguide/tasks.html#automatic-retry-for-known-exceptions